homebytwo/templates/routes/route.html

```diff
@@ -22,29 +17,16 @@
 {% endblock content %}
 
 {% if route.geom %}
-{% block extrajs %}
-<script>
-  function route_map_init(map, options) {
-    const route = L.geoJson({{ route| geojsonfeature | safe }});
-  route.addTo(map);
-  map.fitBounds(route.getBounds());
-  map.dragging.disable();
-  map.scrollWheelZoom.disable();
-  const icon = L.divIcon({
-    className: 'placeIcon placeIcon--default',
-  });
-  const places = Array.from(document.querySelectorAll('.place--known'));
-  const markers = places.map(place => {
-    L.geoJson(JSON.parse(place.dataset.geom), {
-      pointToLayer: (feature, latlng) => L.marker(latlng, {
-        icon: icon,
-        title: feature.properties.name
-      }),
-      style: feature => ({ color: '000000' }),
-      onEachFeature: (feature, layer) => layer.bindPopup(feature.properties.name)
-    }).addTo(map);
-  });
-      }
-</script>
-{% endblock extrajs %}
+  {% block extrajs %}
```

Comment:
We could move that piece of code to a separate template file and include that single file in both templates.

Reply:
So what you are suggesting is having a `{{ include "routes/_leaflet_map.html" }}` in both `_route_map.html` and `_route_edit_map.html`? And in it would be the `div` + the js?

Reply:
I meant just the <script> tag in both `route.html` and `route_form.html`. But whatever prevents the duplication would be ok.